sormas-backend/src/main/java/de/symeda/sormas/backend/caze/Case.java

```diff
@@ -1242,6 +1244,15 @@ public void setExternalToken(String externalToken) {
 		this.externalToken = externalToken;
 	}
 
+	@Column(length = COLUMN_LENGTH_DEFAULT)
```

Comment:
Columns of type text should have `@Column(columnDefinition = "text")` not `@Column(length = COLUMN_LENGTH_DEFAULT)`